metrics/privacy_metrics/dcr_nndr_test.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+import pandas as pd
+import numpy as np
+from metrics.privacy_metrics.dcr_nndr import compare, prepare_common_data_format
+
+original_without_duplicates_1 = pd.DataFrame(
+    {
+        "variable_1": [1,2,9,10],
+        "variable_2": ["Oui", "Oui", "Non", "Non"],
+        "id": [1,2,3,4],
+    }
+)
+
+synthetic_without_duplicates_1 = pd.DataFrame(
+    {
+        "variable_1": [1,2,9,10],
+        "variable_2": ["Oui", "Oui", "Non", "Non"],
+        "id": [1,2,3,4],
+    }
+)
+
+synthetic_without_duplicates_2 = pd.DataFrame(
+    {
+        "variable_1": [3,4,11,12],
+        "variable_2": ["Oui", "Oui", "Non", "Non"],
+        "id": [1,2,3,4],
+    }
+)
```

Comment:
why do you have a `synthetic_without_duplicates_2`? If it is truly needed, I think the naming should change